docs/4.1/admin-guide.md

```diff
@@ -2337,14 +2368,64 @@ When upgrading multiple clusters:
 1. First, upgrade the main cluster, i.e. the one which other clusters trust.
 2. Upgrade the trusted clusters.
 
+## Backing Up Teleport
+
+With Teleport 4.1 you can now quickly export a collection of resources from 
+Teleport. This feature set works best for local and etcd, it's currently experimental
+for AWS/GCP. 
+
+Using `tctl get all` it'll backup. 
```

Comment:
`will backup:` (the colloquial English is a bit confusing/inappropriate for docs)